docs/roman/ramp_fitting/description.rst

```diff
@@ -156,64 +118,80 @@ sufficient; they are given as:
 | 100               |                        | 10       |
 +-------------------+------------------------+----------+
 
-Segment-specific Computations:
-------------------------------
-The variance of the slope of a segment due to read noise is:
+Segment-Specific Computations
+-----------------------------
+
+The segment fitting implementation is based on Section 5 of Casertano et.
+al. 2022. A set of auxiliary quantities are computed as follows:
 
 .. math::
-   var^R_{s} = \frac{12 \ R^2 }{ (ngroups_{s}^3 - ngroups_{s})(group_time^2) } \,,
+   F0 &= \sum_{i=0}^{N-1} W_i
+
+   F1 &= \sum_{i=0}^{N-1} W_i \bar t_i
 
-where :math:`R` is the noise in the difference between 2 frames,
-:math:`ngroups_{s}` is the number of groups in the segment, and :math:`group_time` is the group
-time in seconds (from the exposure.group_time).
+   F2 &= \sum_{i=0}^{N-1} W_i \bar t_i^2
 
-The variance of the slope in a segment due to Poisson noise is:
+The denominator, :math:`D`, is calculated as a single two-dimensional array:
 
 .. math::
-   var^P_{s} = \frac{ slope_{est} }{  tgroup \times gain\ (ngroups_{s} -1)}  \,,
+   D = F2 \cdot F0 - F1^2
 
-where :math:`gain` is the gain for the pixel (from the GAIN reference file),
-in e/DN. The :math:`slope_{est}` is an overall estimated slope of the pixel,
-calculated by taking the median of the first differences of the groups that are
-unaffected by saturation and cosmic rays. This is a more
-robust estimate of the slope than the segment-specific slope, which may be noisy
-for short segments.
 
-The combined variance of the slope of a segment is the sum of the variances:
+The resultant coefficients, :math:`K_i`, are computed as N two dimensional
+arrays:
 
 .. math::
-   var^C_{s} = var^R_{s} + var^P_{s}
+   K_i = (F0 \cdot \bar t_i - F1) \cdot W_i / D
 
+The estimated slope, :math:`\hat F`, is computed as a sum over the resultants
+:math:`R_i` and the coefficients :math:`K_i`:
 
-Exposure-level computations:
-----------------------------
+.. math::
+   \hat F = \sum_{i} K_i R_i
+
+The calculation is skipped for pixels that have :math:`D = 0`. Note that the coefficient
+:math:`K_i` vanishes for each resultant that has a bad pixel, as a consequence of :math:`W_i`
+vanishing.
 
-The variance of the slope due to read noise is:
+The read-noise component :math:`V_R` of the slope variance is computed as:
 
 .. math::
-   var^R_{o} = \frac{1}{ \sum_{s} \frac{1}{ var^R_{s}}}
+   V_R = \sum_{i=0}^{N-1} K_i^2 \cdot (RN)^2 / N_i
 
-where the sum is over all segments.
+The signal variance, :math:`V_S`, of the count rate in the signal-based component of the slope
+variance is computed as:
 
+.. math::
+   V_S = \sum_{i=0}^{N-1} {K_i^2 \tau_i} + \sum_{i<j} {2 K_i K_j \cdot \bar t_i}
 
-The variance of the slope due to Poisson noise is:
+Total variance, if desired, is a (biased) estimate of the total slope variance :math:`V` can
```

Comment:
Let's leave off "biased."  There is a small bias due to the way the weights P are selected, which leads to a bias in the flux and therefore in the total variance.  But if we're going to talk about the biases people care much more about the O(0.1%) bias in the flux than in the O(0.1%) bias in the variance.

Reply:
removed